src/main/java/org/symphonyoss/symphony/messageml/elements/Button.java

```diff
@@ -43,10 +43,19 @@ public void buildAttribute(MessageMLParser parser,
     switch (item.getNodeName()) {
       case NAME_ATTR:
       case TYPE_ATTR:
-      case CLASS_ATTR:
         setAttribute(item.getNodeName(), getStringAttribute(item));
         break;
         // The button can a have tooltips but is not a tooltipable element because it dont generate the span with tooltip
+      case CLASS_ATTR:
```

Comment:
maybe even simpler in a single line?

```java
      case CLASS_ATTR:
        // convert primary-destructive, secondary-destructive -> primary, secondary because the first one is deprecated
        setAttribute(item.getNodeName(), StringUtils.removeEnd(getStringAttribute(item), "-destructive"));
        break;
```

(as in class attr, value ending by "-destrutive" can be only primary, secondary, as they are validated before by validate method. It should work, but I don't have tried...)